README.md

```diff
@@ -2,10 +2,17 @@
 
 HyperSpy extensions are packages that use the hyperspy extension mechanism to add functionality to HyperSpy (such as GUI widgets) and/or register new subclasses of HyperSpy objects (such as components and signals).
 
-The following is a non-exhaustive list of HyperSpy extensions. If you would like to add your package here, just send us a pull request.
+The following is a non-exhaustive list of HyperSpy extensions and the `signal_type` they provide:
 
-| Package name                                                                   | Brief description                                                    |
-|--------------------------------------------------------------------------------|----------------------------------------------------------------------|
-| [hyperspy-gui-ipywidgets](https://github.com/hyperspy/hyperspy_gui_ipywidgets) | ipywidgets widgets for HyperSpy                                      |
-| [hyperspy-gui-traitsui](https://github.com/hyperspy/hyperspy_gui_traitsui)     | traitsui widgets for HyperSpy                                        |
-| [KikuchiPy](https://github.com/kikuchipy/kikuchipy)                            | Processing and analysis of electron backscatter diffraction patterns |
+| Package name                                                                   | Brief description                                                    | `signal_type`                                                    |
+|--------------------------------------------------------------------------------|----------------------------------------------------------------------|------------------------------------------------------------------|
+| [hyperspy](https://github.com/hyperspy/hyperspy)                               | Core library                                                         | `EELS`, `EDS_TEM`, `EDS_SEM`, `DielectricFunction`, `hologram`   |
+| [hyperspy-gui-ipywidgets](https://github.com/hyperspy/hyperspy_gui_ipywidgets) | ipywidgets widgets for HyperSpy                                      |                                                                  |
+| [hyperspy-gui-traitsui](https://github.com/hyperspy/hyperspy_gui_traitsui)     | traitsui widgets for HyperSpy                                        |                                                                  |
+| [KikuchiPy](https://github.com/kikuchipy/kikuchipy)                            | Processing and analysis of electron backscatter diffraction patterns | `EBSD`, `EBSDMasterPattern`, `VirtualBSEImage`                   |
+| [pyXem](https://github.com/pyxem/pyxem)                                        | Multi-dimensional diffraction microscopy                             | `diffraction`, `electron_diffraction`, `diffraction_vectors`, `template_matching`, `vector_matching`, `crystallographic_map`, `vdf_image`, `reduced_intensity`, `diffraction_variance`, `dpc`, `correlation`, `power` |
```

Comment:
Can we keep lowercase only please i.e. pyxem?

I think the list of signal types for the latest release version should be:

`diffraction`, `electron_diffraction`,  `polar_diffraction`, `diffraction_vectors`, `template_matching`, `vector_matching`, `crystallographic_map`,  `tensor_field`,  `vdf_image`,  `reduced_intensity`, `pair_distribution_function`, `diffraction_variance`, `correlation`, `power`

The dpc signals will only be introduced in pyxem-0.12.0 which is not yet released.